README.md

```diff
@@ -35,3 +35,6 @@ The application is developed using flask. Use the following step to setup local
 This project is managed with kustomize.
 Use the following command to deploy on OpenShift cluster.
 `kustomize build manifests/overlays/test | oc apply -f -`
+
+### Latest Updates
+Project is being archived. Read comment [here](https://github.com/thoth-station/core/issues/425#issuecomment-1334012921) for more info on how to use this app.
```

Comment:
It would be great if we copy the details here instead of pointing to the issue.

Reply:
change this to https://github.com/thoth-station/ttm-as-a-service/blob/b62caca5e075bb19b2326b16a7bed9110fd3f5b6/.pre-commit-config.yaml#L37
github url inplace of gitlab